Products/ZenModel/skins/zenmodel/editUserSettings.pt

```diff
@@ -218,6 +218,29 @@
             </script>
           </td>
         </tr>
+        <tr>
+          <td class="tableheader" align="left">Date format</td>
+          <td class="tablevalues">
+            <select id="user_settings_dateformat" name="dateFormat"></select>
+            <script tal:content='string:
+                 Zenoss.env.CURRENT_USER_DATE_FORMAT = "${context/dateFormat}" || Zenoss.USER_DATE_FORMAT;
+             '></script>
+            <script>
+              // build a select list with the current date format selected.
+              // Zenoss.data.timeZones comes from moment-timezone-data.js and
+              Ext.onReady(function() {
+                  var select = Ext.get('user_settings_dateformat').dom, formats = ['MM/DD/YY', 'DD/MM/YY', 'YY/MM/DD'], format, i=0;
+                  for (format of formats) {
```

Comment:
`for...of` is a brand new js feature that will not work on many browsers. Also, you need a counter `i` here, so it makes more sense to use something that provides that. You probably want `Array.forEach` which provides each element and its index in the array.